package.json

```diff
@@ -187,8 +189,8 @@
     "workbox-webpack-plugin": "^6.0.2"
   },
   "engines": {
-      "node": "14.2.0",
-      "npm": "6.14.13"
+    "node": "14.2.0",
+    "npm": "6.14.13"
```

Comment:
not important, but will using `yarn` instead of `npm` make a difference? I know npm can be quite slow

Reply:
Yes, I have never used `npm` over the last couple of years, but when I tried adding material ui libraries, `yarn` had issues, but `npm` worked out suprisingly.

Please let me know which tool lets you successfully install dependencies.